shared/references/cce-sle12-avail.txt

```diff
@@ -1,72 +1,3 @@
-CCE-92227-8
```

Comment:
It seems more CCEs were removed from this list than included in the rule.yml files in this PR. Is that intentional?